### DISCUSSION TITLE
Can you make fighters board ships?

### DISCUSSION THREAD

**Discussion by lumbar527** [endless-sky/endless-sky#8743] [General]
I am working on a plugin and in testing have noticed that fighters are unable to board ships. Is there any way to change that in the ship definition, or are there any alternatives?
I would prefer not to have two variants of each type of ship, as that complicates things.

**Comment by mOctave** [comment]
No, unfortunately, fighters cannot board ships, nor can any bay-based ships.

This makes for some humorous messages when you have, say a "Pocket Medium Warship" with both ship and ship type defined in a plugin, and try to board something with it.

Having this feature you suggest with the ship definition would be a nice solution to this problem, with fighters and drones having an attribute set by default, but with the option to turn it on/off per ship.

**Comment by lumbar527** [comment]
I think what I might do is make another variant that is an interceptor, but keep the fighter being sold, because I have no idea how to program another attribute.
